paasta_tools/cli/cmds/logs.py

```diff
@@ -1362,16 +1478,16 @@ def paasta_logs(args: argparse.Namespace) -> int:
 
     levels = [DEFAULT_LOGLEVEL, "debug"]
 
-    log.debug(f"Going to get logs for {service} on cluster {cluster}")
+    log.debug(f"Going to get logs for {service} on cluster {clusters}")
 
-    log_reader = get_log_reader()
```

Comment:
with the current state only a single cluster is supported, if i get it right, which is enforced by `nargs=1` (won't work with eg `-c cluster1 cluster2` parameter ) at https://github.com/Yelp/paasta/blob/56fac3eaacf701a171ec1f39ddaedf99103ddb6f/paasta_tools/cli/cmds/logs.py#L90-L94

and implicitly mentioned here: https://github.com/Yelp/paasta/blob/56fac3eaacf701a171ec1f39ddaedf99103ddb6f/paasta_tools/cli/cmds/logs.py#L1336-L1339 

This make things much easier tbh, otherwise with multiple clusters we'd need to implement multithreading in vector-logs driver to read logs from two backends in parallel, similar to what scribe has implemented in https://github.com/Yelp/paasta/blob/56fac3eaacf701a171ec1f39ddaedf99103ddb6f/paasta_tools/cli/cmds/logs.py#L599

Reply:
ah, gotcha - I was led astray by the `cluster = args.cluster` -> `cluster` = args.cluster` rename :)

that said, it does say the default is all clusters, so i wonder how that works right now

Reply:
ah, i think that help text is outdated :p